Tg feature campaignmodal

talgrewal opened the pull request:
# Tg feature campaignmodal


- Add Content and style to Campaign modal 
- Add logic to open and close modal
- Add ability to subscribe to campaigns from campaign modal

talgrewal committed:
Add campaign data to more campaign

talgrewal committed:
Add styles to campaign modal

talgrewal committed:
Add Fixed width to title

talgrewal committed:
Merge branch 'develop' into tg-feature-campaignmodal

siddharthparmar7 commented on client/js/screens/CampaignModal/CampaignModal.js:
```diff
@@ -1,8 +1,92 @@
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import styles from '../Campaign/styles';
+import {colors} from '../../config/styles';
+// import SafeAreaView from 'react-native-safe-area-view';
```
Please remove commented code if not needed.

talgrewal replied:
Done

siddharthparmar7 commented on client/js/screens/CampaignModal/CampaignModal.js:
```diff
@@ -1,8 +1,92 @@
-const CampaignModal = props => {
-  return <Text>Event More Screen</Text>;
+const CampaignModal = ({navigation, campaignData, user}) => {
+  console.log(user);
```
remove this console log.

talgrewal replied:
Done

siddharthparmar7 reviewed (changes requested):
Please explain why did you delete client/js/screens/CampaignModal/styles.js file.

siddharthparmar7 commented:
And remove .DS_Store from your branch. Update your .gitignore file to not upload it again.

talgrewal commented:
Writing this in response to your comment above: 
Please explain why did you delete client/js/screens/CampaignModal/styles.js file.

I removed the file because I realized there is tons of css that is duplicate of the campaign page. Tien suggested this on another page but I largly reused css from the campaign page so I deleted the styles files and linked to the other file.

talgrewal committed:
Resolve git merge conflicts

talgrewal committed:
Remove ds store

talgrewal committed:
Resolve git conflict

talgrewal commented:
Ready for review.

siddharthparmar7 commented:
Nice work. Please resolve merge conflicts.

talgrewal committed:
Git merge conflicts

talgrewal committed:
resolve merge conflicts

talgrewal committed:
delete context
